source/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(srcs
+    "m3_api_libc.c"
+    "m3_api_wasi.c"
+    "m3_bind.c"
+    "m3_code.c"
+    "m3_compile.c"
+    "m3_core.c"
+    "m3_emit.c"
+    "m3_env.c"
+    "m3_exec.c"
+    "m3_info.c"
+    "m3_module.c"
+    "m3_optimize.c"
+    "m3_parse.c"
+)
+
+add_library(libwasm3 STATIC ${srcs})
```

Comment:
@vshymanskyy  I'm not sure if this should be `wasm3` or `m3`, please advise.

Note that at the moment the static library is called `liblibwasm3.a`... The reason I didn't use `wasm3` as the target name, is that there is already a `wasm3` target defined in the main cmakelists file.

Reply:
yes m3 is used as file prefix ;) we can include all of the `m3_*.c` files safely

Reply:
so `libm3.a` for the library name and `m3` for the CMake target name is okay?

Reply:
Should be ok!